ProcessLib/Output/GlobalVectorFromNamedFunction.cpp

```diff
@@ -44,11 +44,13 @@ GlobalVector const& GlobalVectorFromNamedFunction::call(
     assert(dof_table.getNumberOfComponents() == static_cast<int>(n_args));
     std::vector<double> args(n_args);
 
-    for (GlobalIndexType node_id = 0; node_id < nnodes; ++node_id) {
-        // TODO maybe fill args via callback mechanism or remove this class entirely.
-        // Caution: The order of args will be the same as the order of the
-        // components in the global vector!
-        for (std::size_t i = 0; i < n_args; ++i) {
+    for (GlobalIndexType node_id = 0; node_id < nnodes; ++node_id)
```

Comment:
Minor: In this commit (`[PL] Move output related files into Output folder.`) there are many (unnecessary) formatting changes, which might cause some trouble for people who have changes in these files in some private branches. Maybe in the future such reformatting could be avoided.

Reply:
I agree, reformatting can be problematic. Currently all the new code since approximately one year is consistently formatted. The older code is still "free" formatted. It's a slow transition....